reana/cli.py

```diff
@@ -1687,15 +1780,16 @@ def cluster_undeploy():  # noqa: D301
     help="Destroy and recreate cluster from scratch?",
 )
 @cli.command(name="run-ci")
-def run_ci(mount, recreate):  # noqa: D301
+def run_ci(mounts, recreate):  # noqa: D301
```

Comment:
Since the mount options are only used during cluster creation time, one cannot easily change mount? do-not-mount? decision once the cluster is created.  So perhaps we can move the recreate command to cluster-create so that we don't have to duplicate all these arguments?

This would mean people will have to do:

```console
$ reana-dev cluster-create
$ reana-dev run-ci
$ reana-dev cluster-delete
``` 

manually.

If it seems OK, then we should do:

- enrich `cluster-create` so that it does docker-pulling and kind-load-docker-image steps after cluster creation;
- leave out cluster-delete step, people will have to do it manually
- remove mount, recreate option from run-ci, because there were consumed already
- enrich run-ci options with --debug flag so that docker images would be built with DEBUG flag
```